developer/app-router/installation.mdx

```diff
@@ -25,8 +26,7 @@ or want to set things up yourself, continue with the rest of this guide.
   <Step title="Open your Next.js project">
     First, open your Next.js project. If you don't already have one, head over to
     the [Next.js](https://nextjs.org/docs/getting-started/installation)
-    documentation to get one set up. If you do have one, please verify you are
-    using [Next.js 13.4](https://nextjs.org/blog/next-13-4) or a later version that is using [App Router](https://nextjs.org/docs/getting-started/installation).
+    documentation to get one set up. If you do have one, please verify you are using [Next.js 13.4](https://nextjs.org/blog/next-13-4) or a later version that is using [App Router](https://nextjs.org/docs/getting-started/installation).
```

Comment:
Looks like we had some formatting changes here but no change in content. We should probably set up Prettier to avoid this happening as different developers with different VS Code settings contribute to the docs.

Reply:
Yeah I noticed that too. I was thinking about doing a PR where I went through and saved each file so that it was auto-formatted.  We do have a prettier file already configured, so after saving each file once, it should be standardized for future changes.

Reply:
@migueloller Are you ok with merging this one and I'll follow up with a separate PR with all the files formatted to avoid this in the future?

Also, I'll follow conventional commits in the future!

Reply:
Yeah, go for it!

And thanks 😄